voice_servers/ts3_manager/egg-t-s3-manager.json

```diff
@@ -22,7 +22,7 @@
     },
     "scripts": {
         "installation": {
-            "script": "apt -y update\r\napt -y upgrade\r\napt install -y git unzip jq wget tar curl\r\n\r\nexport HOME=\/mnt\/server\r\ncd $HOME\r\n\r\n## get release info and download links\r\nLATEST_RELEASE=$(curl -L -s -H 'Accept: application\/json' https:\/\/github.com\/${GITHUB_PACKAGE}\/releases\/latest)\r\nLATEST_VERSION=$(echo $LATEST_RELEASE | sed -e 's\/.*\"tag_name\":\"\\([^\"]*\\)\".*\/\\1\/')\r\n\r\nif [ -z \"${VERSION}\" ] || [ \"${VERSION}\" == \"latest\" ]; then\r\n    DOWNLOAD_LINK=\"https:\/\/github.com\/${GITHUB_PACKAGE}\/releases\/download\/$LATEST_VERSION\/$MATCH-$LATEST_VERSION\"\r\nelse \r\n    DOWNLOAD_LINK=\"https:\/\/github.com\/${GITHUB_PACKAGE}\/releases\/download\/v$VERSION\/$MATCH-$VERSION-v$VERSION\"\r\nfi\r\n\r\necho $DOWNLOAD_LINK\r\nwget $DOWNLOAD_LINK\r\n\r\nif [ -z \"${VERSION}\" ] || [ \"${VERSION}\" == \"latest\" ]; then\r\n    mv -f $MATCH-$LATEST_VERSION $HOME\/start_ts3-manager\r\n    chmod +x start_ts3-manager\r\nelse\r\n    mv -f $MATCH-v$VERSION $HOME\/start_ts3-manager\r\n    chmod start_ts3-manager\r\nfi",
+            "script": "apt update\r\napt -y upgrade\r\n\r\nexport HOME=\/mnt\/server\r\ncd $HOME\r\n\r\n## get release info and download links\r\nLATEST_RELEASE=$(curl -L -s -H 'Accept: application\/json' https:\/\/github.com\/${GITHUB_PACKAGE}\/releases\/latest)\r\nLATEST_VERSION=$(echo $LATEST_RELEASE | sed -e 's\/.*\"tag_name\":\"\\([^\"]*\\)\".*\/\\1\/')\r\n\r\nif [ -z \"${VERSION}\" ] || [ \"${VERSION}\" == \"latest\" ]; then\r\n    DOWNLOAD_LINK=\"https:\/\/github.com\/${GITHUB_PACKAGE}\/releases\/download\/$LATEST_VERSION\/$MATCH-$LATEST_VERSION\"\r\nelse \r\n    DOWNLOAD_LINK=\"https:\/\/github.com\/${GITHUB_PACKAGE}\/releases\/download\/v$VERSION\/$MATCH-$VERSION-v$VERSION\"\r\nfi\r\n\r\necho $DOWNLOAD_LINK\r\ncurl -L -O $DOWNLOAD_LINK\r\n\r\nif [ -z \"${VERSION}\" ] || [ \"${VERSION}\" == \"latest\" ]; then\r\n    mv -f $MATCH-$LATEST_VERSION $HOME\/start_ts3-manager\r\n    chmod +x start_ts3-manager\r\nelse\r\n    mv -f $MATCH-v$VERSION $HOME\/start_ts3-manager\r\n    chmod start_ts3-manager\r\nfi",
```

Comment:
The apt update/upgrade is an unnecessary step that'll only waste time. Install image is rebuilt every Monday. The packages and sources are guaranteed to be up to date.

Reply:
99% of all images have this. and 3 packages are 20 sec. so not really an time issue ;)

Reply:
If egg updates are not desired, then please say so, but please do not demand something that you do not do yourself.